src/helpers/gho-launch/tests/AaveV3GHORemoteLaneTest.sol

```diff
@@ -0,0 +1,338 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPool as IPool_CCIP} from 'src/interfaces/ccip/tokenPool/IPool.sol';
+import {IClient} from 'src/interfaces/ccip/IClient.sol';
+import {IInternal} from 'src/interfaces/ccip/IInternal.sol';
+import {IRateLimiter} from 'src/interfaces/ccip/IRateLimiter.sol';
+import {GhoCCIPChains} from '../constants/GhoCCIPChains.sol';
+import {AaveV3GHOLaneTest} from './AaveV3GHOLaneTest.sol';
+
+abstract contract AaveV3GHORemoteLaneTest_PreExecution is AaveV3GHOLaneTest {
+  constructor(
+    GhoCCIPChains.ChainInfo memory localChainInfo,
+    GhoCCIPChains.ChainInfo memory remoteChainInfo,
+    string memory rpcAlias,
+    uint256 blockNumber
+  ) AaveV3GHOLaneTest(localChainInfo, remoteChainInfo, rpcAlias, blockNumber) {}
+
+  // This test is not necessary but without it the compiler triggers stack too deep... very strange.
+  function test_currentPoolConfig() public virtual {
+    uint64[] memory supportedChains = LOCAL_TOKEN_POOL.getSupportedChains();
+
+    for (uint256 i = 0; i < supportedChains.length; i++) {
+      _assertAgainstSupportedChain(GhoCCIPChains.getChainInfoBySelector(supportedChains[i]));
+    }
+  }
```

Comment:
This is the only thing I am not happy with. This test is not really necessary, but **_if I remove it the compiler throws stack too deep_**. Very strange behavior.

I usually solve stack too deep errors by re-ordering variables or introducing scoping code blocks or structs, but this is a strange case.

I would prefer if we could accept this tech debt for now, so we can proceed with the GHO launch on Ink, and revisit it later with more time.

Flagging it so you guys are aware. Also you might have some suggestions.